deploy/helm/sumologic/conf/setup/setup.sh

```diff
@@ -1,15 +1,23 @@
 #!/bin/bash
-cp /etc/terraform/{locals,main,providers,resources,variables}.tf /terraform
-cd /terraform
 
 # Fix URL to remove "v1" or "v1/"
 export SUMOLOGIC_BASE_URL=${SUMOLOGIC_BASE_URL%v1*}
-
 # Support proxy for terraform
 export HTTP_PROXY=${HTTP_PROXY:=""}
 export HTTPS_PROXY=${HTTPS_PROXY:=""}
 export NO_PROXY=${NO_PROXY:=""}
 
+function remaining_fields() {
+    readonly RESPONSE="$(curl -XGET -s \
+        -u "${SUMOLOGIC_ACCESSID}:${SUMOLOGIC_ACCESSKEY}" \
+        "${SUMOLOGIC_BASE_URL}"v1/fields/quota)"
+
+    echo "${RESPONSE}" | jq '.remaining'
+}
+
+cp /etc/terraform/{locals,main,providers,resources,variables}.tf /terraform
+cd /terraform
```

Comment:
what about `fields.tf`?

Reply:
Done, please check.
I'm not sure where `/etc/terraform` is mounted though so I don't know if copy from `/etc/terraform/fields/` is correct.

Reply:
`firlds.tf` it's not mounted as all as it is in subdirectory
`{{- (tpl (.Files.Glob "conf/setup/*").AsConfig .) | nindent 2 }}`
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/templates/setup/setup-configmap.yaml